src/map/skill.cpp

```diff
@@ -12792,6 +12862,11 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		}
 		break;
 
+	case HN_HELLS_DRIVE:
```

Comment:
Should this really be in skill_castend_nodamage_id?
Or should it be moved to skill_castend_damage_id?

Reply:
Agreed, this can be moved to the generic splash skill list!